nautobot/core/settings.py

```diff
@@ -116,6 +116,8 @@
 # Test runner that is aware of our use of "integration" tags and only runs
 # integration tests if explicitly passed in with `nautobot-server test --tag integration`.
 TEST_RUNNER = "nautobot.core.tests.runner.NautobotTestRunner"
+NUM_SLOW_TESTS = None
```

Comment:
Is this a `django-slowtests` setting? (If so, can you add some comments?) Or can we rename it to something less generic?